<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Backtest</class>
 <widget class="QDialog" name="Backtest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Backtesting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="TableBacktestResult" name="tableBacktestResult"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>回测数据</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="bDatabaseName"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>开始日期</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDateEdit" name="bStartTime">
           <property name="date">
            <date>
             <year>2021</year>
             <month>3</month>
             <day>21</day>
            </date>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>结束日期</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDateEdit" name="bEndTime">
           <property name="date">
            <date>
             <year>2022</year>
             <month>3</month>
             <day>20</day>
            </date>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>回测资金</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="bStartFund">
           <property name="maximum">
            <double>20000000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>1000000.000000000000000</double>
           </property>
           <property name="value">
            <double>10000000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>手续费率</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QDoubleSpinBox" name="bHandlingFeeRate">
           <property name="suffix">
            <string>%</string>
           </property>
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
           <property name="value">
            <double>0.002500000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">QProgressBar{
	background:black;
}
QProgressBar::chunk{
	background:#90CAF9;
}</string>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="textVisible">
            <bool>true</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
           <property name="textDirection">
            <enum>QProgressBar::TopToBottom</enum>
           </property>
           <property name="format">
            <string>当前进度：%p%</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="start">
           <property name="text">
            <string>开始回测</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stop">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>停止回测</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="bCharts">
     <item>
      <widget class="QWebEngineView" name="webEngineView">
       <property name="url">
        <url>
         <string>about:blank</string>
        </url>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
  <customwidget>
   <class>TableBacktestResult</class>
   <extends>QTableView</extends>
   <header>TableBacktestResult.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
